proto/proto/store.proto

```diff
@@ -0,0 +1,93 @@
+// Specification of the database RPC.
+//
+// This provided access to the rollup data to the other nodes.
+syntax = "proto3";
+package store;
+
+import "account.proto";
+import "block.proto";
+import "digest.proto";
+import "note.proto";
+import "transaction.proto";
+
+// REQUESTS
+// ================================================================================================
+
+message GetBlockHeaderRequest {
+    block.BlockSpec block_spec = 1;
+}
+
+message GetNoteRequest {
+    note.NoteSpec note_spec = 2;
+}
+
+message GetTransactionDataRequest {
+    account.AccountId account_id = 1;
+    block.BlockSpec block_spec = 2;
+    repeated note.NoteOrigin notes = 3;
+}
+
+message ListNotesByTagRequest {
+    // Only include tags after this block.
+    block.BlockSpec block_spec = 1;
+    // Minimum value of the tag.
+    fixed64 tag_min = 2;
+    // Maximum value of the tag.
+    fixed64 tag_max = 3;
+}
+
+message SubmitTransactionRequest {
+    transaction.ProvenTransaction transaction = 1;
+}
```

Comment:
The two transaction-related messages could probably be removed from here.

Reply:
done